docker/base

```diff
@@ -25,11 +25,18 @@ apt -y install apt-transport-https build-essential bzip2 devscripts \
 git config --system user.name "Cachalot Docker image"
 git config --system user.email "tsunami@sociomantic.com"
 
-# Add extra APT repositories from Sociomantic in Bintray
+# Add extra APT repository from dlang-community in Bintray
+dist="$(lsb_release -cs)"
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61
+
+echo '# dlang-community repos' \
+        > /etc/apt/sources.list.d/dlang-community.list
+echo "deb https://dl.bintray.com/dlang-community/apt $dist prerelease release" \
```

Comment:
The prerelease shouldn't be included for now, as we don't have `apt-preferences` properly configured not to pick the prereleases automatically. We should add prereleases at some point, but we need to make sure that they are properly configured.